tests/functional/cylc-play/06-warnif-scp-after-fcp.t

```diff
@@ -53,35 +51,3 @@ for SCP in 1 2 3; do
 done
 
 purge
-
-# Gregorian Cycling
```

Comment:
No need to check this for different cycling types; it should be the responsibility of the `tests/unit/cycling` and `tests/functional/cyclers` tests to ensure cycle point comparison works